.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: required
+language: go
+services:
+  - docker
+install: true
+
+jobs:
+  include:
+    - stage: test
+      script: make test
+    - script: make cbuild
```

Comment:
Is this a separate stage with no name?

Reply:
Unnamed stages inherit the previous stage's name, so you can have multiple simultaneous jobs.